transformFrameId: "/test_frame"
publishingRate: 500.0

